ggmolvis/ggmolvis.py

```diff
@@ -230,23 +272,23 @@ def molecule(self,
             The color of the molecule. Default is 'default'
         material: str
             The material of the molecule. Default is 'default'
-        lens: float
-            The focal length of the camera associated with the molecule.
 
         Returns:
         --------
         molecule: Molecule
             The created `Molecule` object
         """
-        molecule = Molecule(atomgroup=universe.atoms,
+        if isinstance(universe, mda.Universe):
```

Comment:
I'm not familiar enough with MDA universes, but why this line to get the `.atoms` instead of using the universe?`

Reply:
The `Molecule` class (I plan to rename it to Trajectory for consistency) currently assumes the input is an `AtomGroup`. this converts a `Universe` to an `AtomGroup`, just to allow the user to provide either.

Thinking it over, it makes more sense to modify the Molecule class itself to handle both input types and perform the conversion internally.